test/integration2/schema/misc.spec.js

```diff
@@ -543,19 +551,20 @@ describe('Schema (misc)', () => {
         });
 
         describe('enum - postgres', () => {
-          if (!isPgBased(knex)) {
-            return Promise.resolve();
-          }
+          afterEach(async () => {
+            if (!isPgBased(knex)) {
```

Comment:
```suggestion
            if (isPgBased(knex)) {
```
Wrong logic here. We must run when it is PG based